BuildDetails.md

```diff
@@ -0,0 +1,57 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how projects are built and why they are built that way. Please note that thes guide only fully applies to the GameStartr games, although some of it can be applied to the modules as well.
+
+### Table of Contents
+
+1. [How to Build](#how-to-build)
+2. [Grunt Tasks](#grunt-tasks)
+
+## How to Build
+There are two ways to build a FullScreenShenanigans project:
```

Comment:
I don't like this phrasing. It makes it sound like there are two separate ways to build projects, which is incorrect. There is one way that has multiple steps, and you can do all at once or individually.